packages/blue-sdk/src/addresses.ts

```diff
@@ -172,7 +172,8 @@ export const addressesRegistry = {
     metaMorphoFactory: "0xa9c87daB340631C34BB738625C70499e29ddDC98",
     chainlinkOracleFactory: "0x1ff7895Eb842794c5d07C4c547b6730e61295215",
     preLiquidationFactory: "0xeDadDe37D76c72b98725614d0b41C20Fe612d304",
-
+    // Must implement USDC permit version 2 (otherwise breaks permit signatures).
+    usdc: "0x3c499c542cEF5E3811e1192ce70d8cC03d5c3359",
     wNative: "0x0d500B1d8E8eF31E21C99d1Db9A6444d3ADf1270",
```

Comment:
```suggestion

    wNative: "0x0d500B1d8E8eF31E21C99d1Db9A6444d3ADf1270",
    // Must implement USDC permit version 2 (otherwise breaks permit signatures).
    usdc: "0x3c499c542cEF5E3811e1192ce70d8cC03d5c3359",
```